modules/vector-search/partials/nav.adoc

```diff
@@ -1,6 +1,7 @@
 * xref:7.6@server:vector-search:vector-search.adoc[]
 ** xref:7.6@server:vector-search:create-vector-search-index-ui.adoc[]
 ** xref:7.6@server:vector-search:create-vector-search-index-rest-api.adoc[]
+** xref:7.6@server:vector-search:fine-tune-vector-search.adoc[]
```

Comment:
I would move this below all of the Run a Vector Search pages in the TOC, personally.